src/main/kotlin/io/quarkus/code/rest/OpenAPIFilter.kt

```diff
@@ -15,7 +15,15 @@ class OpenAPIFilter : OASFilter {
             if (pathItem.get.operationId.equals("extensionsForStream", ignoreCase = true)) {
                 val parameters = pathItem.get.parameters
                 for (parameter in parameters) {
-                    if (parameter.name.equals("stream", ignoreCase = true)) {
+                    if (parameter.name.equals("streamKey", ignoreCase = true)) {
+                        val schema = parameter.schema
+                        schema.enumeration = ArrayList<Any>(validStreamValues)
+                    }
+                }
+            }else if (pathItem.get.operationId.equals("downloadForStream", ignoreCase = true)) {
```

Comment:
```suggestion
            } else if (pathItem.get.operationId.equals("downloadForStream", ignoreCase = true)) {
```
(not sure we check the formatting of Kotlin files though with our current setup)

Reply:
Ah, I missed it was not in the core repo, forget about the noise :).